### PR TITLE
Batch 2 Updates

### PR DESCRIPTION
**Pull request by mclark58**
Edited display.yaml file to reflect the documentation changes.

### CODE REVIEW

**Comment by mclark58** [comment]
@jayrbolton / @tianhao-gu Not really sure who to tag here. Please merge the PR and push to dev. I will do a final check and then let you know when it's ready to be pushed to prod. Thanks!

File: ui/narrative/methods/runFastQC/display.yaml [hunk 1]
```diff
@@ -31,44 +31,33 @@ suggestions:
...
+      <li><b>Sequence Duplication Levels:</b> counts the degree of duplication for every sequence in a library and creates a plot showing the proportion of sequences with different degrees of duplication (in blue) and de-duplicated sequences (in red).</li>
+      <li><b>Overrepresented Sequences:</b> lists all of the sequence which make up more than 0.1% of the total. An overrepresented sequence implies that either it is highly biologically significant, the library is contaminated, or is not that diverse.</li>
+      <li><b>Adapter Content:</b> displays the proportion of sequences which have an adapter sequence at a given position. This is informative in deciding whether there is a significant amount of adapter present in the sequences and can be subjected to trimming.</li>
+      <li><b>Kmer Content:</b> displays relative K-mer (k=7) enrichment over the read length for the top six K-mers. More specifically, it measures the number of each 7-mer at each position in the library and then uses a binomial test to look for significant deviations from an even coverage at all positions and reports the 7-mers with positionally biased enrichment.</li>
```
Comment (jayrbolton) [inline ui/narrative/methods/runFastQC/display.yaml:54]: Add closing `</ul>` tag here

File: ui/narrative/methods/runFastQC/display.yaml [hunk 2]
```diff
@@ -31,44 +31,33 @@ suggestions:
...
+    <p>FastQC is a quality control application that allows users to perform numerous quality control checks on raw sequence data generated by high throughput sequencing pipelines such as Illumina and ABI SOLiD platforms in FASTQ format. It generates as output a comprehensive multi-page report on the composition and quality of reads in HTML format, with one page for each of the reads (e.g. Single End, Paired End: forward, Paired End: reverse). The report can be viewed inside the Narrative or as a new web page that can also be downloaded.</p>
 
-    <p>The HTML report shows a summary of the modules that were run by FastQC, and provides a quick assessment of the quality of the results. The result legends differentiate among normal (green tick), slighly abnormal (orange exclamation), and very unusual (red cross) reads.</p>
+    <p>The HTML report includes results from multiple modules that were run by FastQC, and provides a quick assessment of the quality of the results labeled as normal (green checkmark), slightly abnormal (orange triangle), and very unusual (red cross) reads. The modules included in the report are as follows:
```
Comment (jayrbolton) [inline ui/narrative/methods/runFastQC/display.yaml:42]: Add closing `</p>` tag here

**Review by jayrbolton — CHANGES_REQUESTED** [review]
just a couple tags to close

**Comment by jayrbolton** [comment]
lol yall never closed the tags

**Comment by jayrbolton** [comment]
nvm i see paramvir did it on master.

**Pull request merged**